components/internal/RequestStatusBadge.tsx

```diff
@@ -15,110 +15,118 @@ export default function RequestStatusBadge({ variant }: Props) {
     switch (variant) {
       case 'COMPLETED':
         return (
-          <>
-            <Image
-              src="/assets/completed-icon.svg"
-              alt="Completed Application Icon"
-              height={12}
-              width={12}
-            />
-            <Box as="span" ml="2">
-              Completed
+          <Badge variant={variant} width="127px" height="32px" paddingX="12px" paddingY="4px">
```

Comment:
The `RequestStatusBadge` seem to sometimes resize based on the container they're in so I refactored it and specified the height and width explicitly

Reply:
Good catch! I think a simpler fix for this might have been to leave this file unchanged, except change the last return statement to this:

```
  return (
    <Wrap>
      <Badge variant={variant}>
        <Box ml="-4px" mr="8px">
          {_renderBadgeContent(variant)}
        </Box>
      </Badge>
    </Wrap>
  );
```

The only difference between this and your original code is the `<Wrap></Wrap>`, and this seemed to fix the issue as well. Feel free to stick to either solution, although this one would eliminate some of those hardcoded widths.

Reply:
That looks cleaner so I implemented your suggestion!

I noticed that some positioning/spacing was slightly off so I tweaked the padding a bit to match the designs more closely
Before:
![image](https://user-images.githubusercontent.com/51224641/128812743-4589ec13-27c8-4b23-b500-28c8abc9b03e.png)

After minor padding tweaks (also modified one line from `badge.ts`):
<img width="212" alt="Expiring  30 days" src="https://user-images.githubusercontent.com/51224641/128812268-6b5578ce-f271-436a-9caa-aa033c09854b.png">